examples/1App_OpenMC_VHTR/README.md

```diff
@@ -18,4 +18,4 @@ This example provides a demonstration on how to use WATTS to perform a single SA
 
 - [__watts_exec.py__](watts_exec.py): WATTS workflow for this example. This is the file to execute to run the problem described above.
 - [__openmc_template__](openmc_template.py): OpenMC templated model.
-
+- [__watts_comprehensive.son__](watts_comprehensive.son): Workbench input file for this example.
```

Comment:
change names of the files - `watts_comprehensive.son` is just an example of input with all options listed out for testing. This file should be more `watts_wb.son`